src/com/hapgood/refreshable.cljc

```diff
@@ -130,6 +184,12 @@
 
 (def close! async/close!)
 
+(def closed? impl/closed?)
+
+(defn refresh! [refreshable]
```

Comment:
There is a race condition in here, which, IIRC, is why it was not exposed initially.

Reply:
Is the race condition you are referring to that it might not be refreshed by the next call?

If so, I had a potential idea for a `refresh!!` in Clojure by adding a temporary watch and blocking on that watch to be fulfilled.